openerp/addons/base/security/base_security.xml

```diff
@@ -44,6 +43,7 @@
         </record>
         <record model="res.groups" id="group_user">
             <field name="implied_ids" eval="[(4, ref('group_no_one'))]"/>
+            <field name="users" eval="[(4, ref('base.user_root'))]"/>
```

Comment:
Why was it working before without this change - you're only recomputing *less often*? Or do you mean it was broken before?